src/components/auth/registration-form.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+
+import * as z from "zod";
+
+import { SubmitHandler, useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import React from "react";
+import Link from 'next/link'
+import { signIn } from 'next-auth/react'
+
+const formSchema = z.object({
+  username: z.string().nonempty("E-mail je povinný").email(),
+  password: z.string().nonempty("Heslo je povinné").min(6, "Heslo musí být dlouhé alespoň 6 znaků"),
+  repeatPassword: z.string().nonempty("Heslo je povinné").min(6, "Heslo musí být dlouhé alespoň 6 znaků"),
+});
+
+
+export type FormValues = z.infer<typeof formSchema>;
+
+export const RegistrationForm = () => {
+  const form = useForm<FormValues>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      username: "",
+      password: "",
+      repeatPassword: "",
+    },
+  });
+
+  const handleSubmit: SubmitHandler<FormValues> = async ({ username, password }, event) => {
+    event?.preventDefault();
+    console.log({ username, password })
+  };
+
+  const errors = form.formState.errors;
+
+  return (
+    <form onSubmit={form.handleSubmit(handleSubmit)} className="space-y-8">
+      <div className="flex min-h-full flex-1 flex-col justify-center px-6 py-12 lg:px-8">
+        <div className="sm:mx-auto sm:w-full sm:max-w-sm">
+          <img
+            className="mx-auto h-10 w-auto"
+            src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=600"
+            alt="Your Company"
+          />
+          <h2 className="mt-10 text-center text-2xl font-bold leading-9 tracking-tight text-gray-900">
+            Registrovat se
```

Comment:
Co se jakýchkoliv textů týká, tak bych to jel přes slovník a jako nice to have bych rozjel nějakou administraci pro texty, aby si to měnili sami. Texty se budou měnit, nechtěl bych při každé změněně prohledávat celý projekt...
@jakub-sedlacek ?

Reply:
Nastavit i18n bychom určitě měli, chtěl jsem to nadhodit na příštím dev meetingu. Administrace se mi zdá jako dost práce navíc, kterou po nás navíc nikdo nepožaduje.

Reply:
Založím task. S administrací bych nedivočil. To je nad scope projektu. Případně by se dal použít nějaký plugin ve WP, ale i to je množství práce, které aktuálně vnímám jako nad rámec scope.